lib/reversetunnel/transport.go

```diff
@@ -246,10 +246,32 @@ func (p *transport) start() {
 			p.log.Debugf("Forwarding connection to %q", p.kubeDialAddr.Addr)
 			servers = append(servers, p.kubeDialAddr.Addr)
 		}
+	// Connect to Windows Desktop service.
+	case LocalWindowsDesktop:
```

Comment:
Seems to be the same logic as below, why not just do `case LocalNode, LocalWindowsDesktop`?

IIRC app/db service connections are handled by the LocalNode case below, you probably wouldn't even need this if desktop service used `reversetunnel.LocalNode` To address when dialing like they do.